app/lib/active_model/errors_to_xml.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module ActiveModel
```

Comment:
Code looks good. I would only suggest to move that module into the 3scale namespace instead of polluting the `ActiveModel` one.

Reply:
Done in https://github.com/3scale/porta/pull/3869/commits/9e7077d8b852c64aab64d08b5178023e46a84000